src/protocol/http/org/apache/jmeter/protocol/http/curl/BasicCurlParser.java

```diff
@@ -194,9 +194,9 @@ public void setCompressed(boolean compressed) {
          * @param value the value of Header
          */
         public void addHeader(String name, String value) {
-            if (name != null && name.equalsIgnoreCase("COOKIE")) {
+            if (name.equalsIgnoreCase("COOKIE")) {
```

Comment:
Felix meant "COOKIE".equalsIgnoreCase(name) which will avoid NPE since "COOKIE" is not null